core/src/Revolution/Processors/Workspace/Lexicon/ReloadFromBase.php

```diff
@@ -41,42 +42,53 @@ public function getLanguageTopics()
      */
     public function process()
     {
-        $language = 'en';
-        $namespace = 'core';
-        $corePath = $this->modx->getOption('core_path', null, MODX_CORE_PATH);
+        $namespace = htmlentities($_POST['namespace']);
```

Comment:
Processors should NOT use $_POST (or other superglobals). Instead, use `$this->getProperty('namespace')`.